azerothcore/azerothcore-wotlk
# fix(Core/Player): Display random properties in inspect

* https://github.com/TrinityCore/TrinityCore/commit/5b7b21c362f8127e98e63e939a419956bc188e20

Co-Authored-By: Paraciusz <paraciusz@users.noreply.github.com>





## CHANGES PROPOSED:
-  
-  


## ISSUES ADDRESSED:
- Closes 



## TESTS PERFORMED:
- Builds
- Works in-game



## HOW TO TEST THE CHANGES:

1. 2 players
2. add item 25247 to one char (Bow)
3. See that you (don't) see the stats and full name when you inspect the character (without the PR)

## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
